hdk-rust/src/api.rs

```diff
@@ -564,7 +563,7 @@ pub fn call<S: Into<String>>(
     })
 }
 
-/// Prints a string through the stdout of the running service, and also
+/// Prints a string through the stdout of the running conductor, and also
```

Comment:
```suggestion
/// Prints a string through the stdout of the running Conductor, and also
```

because we want to make clear as often and consistently as possible that Conductor is a thing we made up